packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -546,6 +561,82 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
       }
     }
 
+    // Expand/Collapse options
+    if (isExpandableGridModel(model) && model.isRowExpandable(modelRow)) {
+      // If there are grouped columns, then it is a rollup
+      // For rollups, the column number will be the depth minus one
+      let cellValue;
+      const colForRollUp = model.depthForRow(modelRow) - 1;
+      const valForRollUp = model.valueForCell(colForRollUp, modelRow);
+      const valForTreeTable = model.valueForCell(0, modelRow);
+      if (model.groupedColumns.length > 0) {
+        if (valForRollUp === null || valForRollUp === undefined) {
+          cellValue = 'null';
+        } else {
+          cellValue = model.textForCell(colForRollUp, modelRow);
+        }
+      } else if (valForTreeTable === null || valForTreeTable === undefined) {
+        cellValue = 'null';
+      } else {
+        cellValue = model.textForCell(0, modelRow);
+      }
```

Comment:
Here we're getting the value for both the rollup and treetable, when we only need the one we're interested in. Also, we have the same logic once we do get that value (if it's null, use `'null'`, otherwise use the `textForCell`). So you could simplify this quite a bit, to something like this:
```
      const expandingColumn = model.groupedColumns.length > 0 ? model.depthForRow(modelRow) - 1 : 0;
      const cellValue = model.valueForCell(expandingColumn, modelRow)
      const cellText =  cellValue == null ? 'null' : model.textForCell(expandingColumn, modelRow);
```